modules/client/examples.md

```diff
@@ -446,6 +446,45 @@ console.log(balances);
 */
 ```
 
+### Ensure an allowance is enouch
```

Comment:
```suggestion
### Ensure a minimum token allowance
```

Reply:
Follow-up question. If the actual contract method is `approve()`, shouldn't the whole thing be called like `ensureMinApproval()`?

Reply:
Mmm, this can lead into confusion imo, we have som parameter on the proposal that is called `minApproval`, also when you check the actual value in the contract you use `allowance` method so ensuring an allowance kinda makes sense, if i had to change it I would change it to `updateAllowance`or something similar

Reply:
mmm multisig-wise that's a very good point, but approving a transfer is ERC20-related, nothing to do with a multisig. If reading is done through `allowance()` then `ensureAllowance()` feels good, because "update" would set it no matter what, whereas "ensure" denoted a "lazy" operation